frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -1665,7 +1668,8 @@ void Visitor::checkAttributeAppliedToCorrectNode(const Attribute* attr) {
     if (attr->name() == USTR("assertOnGpu") || attr->name() == USTR("gpu.assertEligible")) {
       CHPL_REPORT(context_, InvalidGpuAssertion, node, attr);
     } else {
-      CHPL_ASSERT(attr->name() == USTR("gpu.blockSize"));
+      CHPL_ASSERT(attr->name() == USTR("gpu.blockSize") ||
+                  attr->name() == USTR("gpu.itersPerThread"));
       CHPL_REPORT(context_, InvalidBlockSize, node, attr);
```

Comment:
The error is called `InvalidBlockSize`, which is technically not accurate for `gpu.itersPerThread`; I think this problem might predate your PR and doesn't need to be fixed here.

Reply:
Does not look like the problem predates this PR. I am reluctant to fix it though -- because that would entail yet another verbatim copy of the error class, like `InvalidGpuAssertion` is.

Reply:
I was thinking of actually combining the errors into an `InvalidGpuAttribute` error, and reducing duplication.